src/entrypoints/app.ts

```diff
@@ -1,9 +1,6 @@
-// Load polyfill first so HTML imports start resolving
```

Comment:
Someone sorted the imports at some point. The polyfill was meant to be the first import here hence the comment/eslint disable 😆

Reply:
Wasn't me... 🙈 it was `Organize imports`